src/main/java/com/github/alex1304/jdash/util/Routes.java

```diff
@@ -11,7 +11,10 @@ private Routes() {
 	
 	public static final String GET_USER_INFO = "/getGJUserInfo20.php";
 	public static final String USER_SEARCH = "/getGJUsers20.php";
+	public static final String USER_SCORES = "/getGJScores20.php";
+	public static final String GET_USER_COMMENT = "/getGJAccountComments20.php";
 	public static final String DOWNLOAD_LEVEL = "/downloadGJLevel22.php";
+	public static final String GET_LEVEL_COMMENT = "/getGJComments21.php";
```

Comment:
Just add a S to the word "COMMENT", as it fetches more than one. Won't complain too much here as other constant names don't follow a particular logic

Reply:
Added "_s_" to the word "_COMMENT_"